src/color.rs

```diff
@@ -0,0 +1,72 @@
+use crate::{
```

Comment:
This file should be inside desktop as it is specific to there. Similar to say HandleToken and so on